### DISCUSSION TITLE
Latest version not available on docker

### DISCUSSION THREAD

**Discussion by OnlyC0RE** [mxpv/podsync#724] [General]
When I do this:
`docker pull ghcr.io/mxpv/podsync:latest`
I get:
`Error response from daemon: manifest unknown`

mxpv/podsync:latest works but is on version 2.5.0

**Comment by qjiang888** [comment]
Me too, I can't pull the latest image

**Comment by c0nsaw** [comment]
Same

**Comment by mxpv** [comment]
hmm, that's weird. `:latest` should point to `v2.7.0` according to github:
<img width="250" alt="image" src="https://github.com/mxpv/podsync/assets/865334/20678646-50ba-4757-b544-defbfc2d794c">
